image_name: 3netest.pgm
image_header: 4
width: 555
height: 517
resolution: 0.05000
origin: [-13.15, -2.0]
free_value: 255
robot: roberto
